__tests__/app/components/playList.test.tsx

```diff
@@ -0,0 +1,57 @@
+import PlayList from '../../../app/components/playList';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import IPlaylist from '@/app/playlist/iPlaylist';
+import { faker } from "@faker-js/faker";
+import { getPlayLists } from '../../../app/lib/api-service';
+
+jest.mock('../../../app/lib/api-service', () => ({
+    getPlayLists: jest.fn()
+}));
+
+jest.mock("../../../app/components/gridAction", () => () => {
+    return <td />;
+});
+
+describe('playlist', () => {
+    afterAll(() => {
+        jest.clearAllMocks();
+    });
+
+    const playLists: Array<IPlaylist> = [{ 
+        id: faker.string.alphanumeric(),
+        name: faker.string.alphanumeric(), 
+        movies: [faker.string.alphanumeric()]
+    },{ 
+        id: faker.string.alphanumeric(),
+        name: faker.string.alphanumeric(), 
+        movies: [faker.string.alphanumeric()]
+    }];
+
+    (getPlayLists as jest.Mock).mockResolvedValue(playLists);
+
+    it('should contain title playlist', async () => {
+        const component = await PlayList();
+        render(component);
+
+        const element = screen.getByRole('heading');
+        expect(element).toHaveTextContent('PlayList');
+    })
+
+    it('should contain table elements', async () => {
+        const component = await PlayList();
+        render(component);
+
+        const tableElement = screen.getByRole('table');
```

Comment:
Very specific test there can be more then 2 tables in file

Reply:
added data-testid on the table